### PR TITLE
Feature/support ie11

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Blocked By
- [x] no transpiling of js-owncloud-client - https://github.com/owncloud/owncloud-sdk/pull/199
- [x] dropzone fixed
- [x] ~login redirect leads to download of authorize endpoint :roll_eyes:~ only on one pc :man_shrugging: 
- [x] login screen: logo is off - https://github.com/owncloud/owncloud-design-system/issues/313
- [ ] Menu button and avatar with name are not vertically aligned - https://github.com/owncloud/owncloud-design-system/issues/314
- [ ] Search input in topbar has wrong width - https://github.com/owncloud/owncloud-design-system/issues/315
- [ ] Icons are disappearing after hovering them or hovering the name of the file/folder - only in component oc-file - https://github.com/owncloud/owncloud-design-system/issues/316
- [ ] console error - SCRIPT5022: InvalidStateError - https://github.com/owncloud/owncloud-sdk/issues/236
- [x] file upload: [Vue warn]: Error in v-on handler: "ReferenceError: 'find' is undefined"
- [x] remove notification: [Vue warn]: Error in v-on handler: "TypeError: Object doesn't support property or method 'remove'"
- [x] file download not working - no console message - network call is sucessful :man_shrugging: 

ie-login

image

### CODE REVIEW

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2465/

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2632/

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUITags</strong> failed. Please find the screenshots inside ...</summary><p>























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIAdminSettings</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUITags</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingAcceptShares</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRenameFolders</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIUpload</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingNotifications</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingPublic</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingExternal</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIMoveFilesFolders</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUITrashbin</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRenameFiles</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIComments</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUILogin</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIWebdavLocks</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFavorites</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalGroups</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIWebdavLockProtection</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRestrictSharing</strong> failed. Please find the screenshots inside ...</summary><p>
























































</p></details>

File: index.html [hunk 1]
```diff
@@ -7,7 +7,8 @@
 	<meta name="theme-color" content="#375f7E"/>
 	<meta http-equiv="x-ua-compatible" content="IE=11">
 	<link rel="manifest" href="manifest.json">
-  <script src="node_modules/requirejs/require.js"></script>
+	<script src="node_modules/requirejs/require.js"></script>
+	<script src="node_modules/wicked-good-xpath/dist/wgxpath.install.js"></script>
```
Comment (DeepDiver1975) [inline index.html]: can't we add this to webpack?

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary><p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary><p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary><p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary><p>





</p></details>

File: index.html (hunk 1 above)
Comment (DeepDiver1975) [reply index.html]: https://github.com/owncloud/phoenix/issues/1208

**Pull request merged**